lib/mobilize_america_client/client/attendances.rb

```diff
@@ -1,13 +1,17 @@
 module MobilizeAmericaClient
   class Client
     module Attendances
-      def organization_attendances(organization_id:, updated_since: nil, page: nil, per_page: nil)
+      def organization_attendances(organization_id:, updated_since: nil, cursor: nil, page: nil, per_page: nil)
         params = {}
 
         unless updated_since.nil?
           params[:updated_since] = updated_since.to_i
         end
 
+        unless cursor.nil?
+          params[:cursor] = cursor
+        end
+
         unless page.nil?
           params[:page] = page
         end
```

Comment:
What happens when you include both, the `cursor` and `page` params? Should we validate that only one parameter is included?

Reply:
It doesn't seem like validation is the gem's responsibility. Up to the caller and their API.